features/steps/steps.py

```diff
@@ -168,12 +167,15 @@ def when_i_configure_uaclient_using_proxy_machine(
         DEFAULT_CONFIG_FILE, tmp_local_conf
     )
     cfg = yaml.safe_load(load_file(tmp_local_conf))
+    new_cfg = {}
```

Comment:
Interesting, was something failing because of appending the whole config?

Reply:
Yeah, when we run the code twice, we were appending the whole config all over again :(

Reply:
True, I just didn't expect that to break anything - regardless, this implementation is better and cleaner :+1: